app/models/katello/repository.rb

```diff
@@ -90,7 +90,10 @@ class Repository < Katello::Model
       :allow_blank => false,
       :message => ->(_, _) { _("must be one of the following: %s") % Katello::RepositoryTypeManager.repository_types.keys.join(', ') }
     }
-    validates :download_policy, inclusion: { in: ::Runcible::Models::YumImporter::DOWNLOAD_POLICIES }, if: :yum?
+    validates :download_policy, inclusion: {
+      in: ::Runcible::Models::YumImporter::DOWNLOAD_POLICIES,
+      :message => ->(_, _) { _("must be one of the following: %s") % ::Runcible::Models::YumImporter::DOWNLOAD_POLICIES.join(', ') }
```

Comment:
I used the lambda because it followed the format used in the message above this one. I actually didn't realize I was mixing hash syntaxes, I'll fix that.